packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -66,8 +67,8 @@ export abstract class SharedObjectCore<
 		return this;
 	}
 
-	private readonly opProcessingHelper: SampledTelemetryHelper;
-	private readonly callbacksHelper: SampledTelemetryHelper;
+	private readonly opProcessingHelper: SampledTelemetryHelper<ProcessTelemetryProperties>;
+	private readonly callbacksHelper: SampledTelemetryHelper<ProcessTelemetryProperties>;
```

Comment:
I think only the `opProcessingHelper` needs to have the extra properties and thus the generic type. And we're not passing any defaults to `callbacksHelper` below, nor incrementing metrics when it is used, so it doesn't need the generic type, right?

Reply:
Yes. I can remove the generic types from the `callbacksHelper`.